pkg/daemon/config/config.go

```diff
@@ -93,6 +93,8 @@ type Configuration struct {
 	NeighGCThresh1 int
 	NeighGCThresh2 int
 	NeighGCThresh3 int
+
+	EnableVlanArpEnhancement bool
```

Comment:
I thinks this enhancement is working for both vlan and bridge.
How about rename this to `EnableArpEnhancedAddress`?